test/models/comment_test.rb

```diff
@@ -1,7 +1,38 @@
 require 'test_helper'
 
 class CommentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  def test_required_attributes_for_creation
+    new_comment = Comment.new
+
+    assert(!new_comment.valid?)
+    assert(new_comment.errors.messages[:text].present?)
+    assert(new_comment.errors.messages[:report].present?)
+    assert(new_comment.errors.messages[:user].present?)
+
+    user = users(:daniel)
+    report = reports(:first_report)
+
+    new_comment.text = "Lorem"
+    new_comment.user = user
+    new_comment.report = report
+
+    assert(new_comment.valid?)
+    new_comment.save!
+  end
+
+  def test_user_and_report_comment_count
```

Comment:
I don't think test is really necessary. The previous test you made sure that a comment can be persisted as long as you pass the right attributes but in this text you are just checking if the comments count change when a comment is added. This test doesn't add a lot of value, it just tests Rails' ability to save and increment the count when something is saved.